src/webpage/src/app.rs

```diff
@@ -58,34 +61,6 @@ impl Default for App {
 }
 
 impl App {
-    
-    fn reconnect(&mut self) {
```

Comment:
This function is used by process_mavlink_websocket

Reply:
yep, but process_mavlink_websocket is changed not to use it in the next commit:

`src: webpage: src: Refactor message and hub message stats receiving` (ed08a74a53554e5ea1fdaa23b26f385bb5c7cfab)